### PR TITLE
30 adding user info to database

### PR DESCRIPTION
**Pull request by baharbazargan62**
## Naming Rules

#30/Adding user info to database /bahar

adding recaptcha

## Related to

#37

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

**Review by EneVladMihai — COMMENTED** [review]
Looks good to me. Only suggestion from UX perspective would be not to show the ReCAPTCHA and form at the same time. We can show the form first, let them fill the data and when they click the "Next" button we show the ReCAPTHA and hide the form

**Review by haroon-ali-dev — APPROVED** [review]
Everything works great locally. ReCaptcha barrier works, user info is inserted into database correctly. All validation for front-end and back-end works fine.

File: server/routes/user.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          body.dateOfBirth,
+          body.currentLocation,
+          body.pinCode,
+          body.phoneNumber,
+          body.qualification,
+          body.dateOfRelease,
+          body.caseStatus,
+        ]
+      );
+      res.json(result.rows[0]);
```
Comment (EneVladMihai) [inline server/routes/user.js:29]: After successfully inserting something in the DB, it's standard to return a status of `201`.
Making this change will prob have an impact on this [line](https://github.com/Miladepour/SyntaxSquad-Project-Kunji/pull/37/files#diff-d2e54251fe00b6c1aa216984652dbe359d580b1379364133310be76e0250289bR66) as well.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201

File: client/src/components/UserForm.js [hunk 2]
```diff
@@ -204,7 +239,14 @@ export default function UserForm() {
           </Form.Control.Feedback>
         </div>
       </Form.Group>
-
+      <div className="w-50">
+      <ReCAPTCHA
+       sitekey="6LfUTRElAAAAAGygougsf9-TgpDcXrONCKzZGqJP"
```
Comment (EneVladMihai) [inline client/src/components/UserForm.js:244]: This should be stored in an application variable. (e.g. the ones that start with `REACT_APP_`).

[See doc](https://create-react-app.dev/docs/adding-custom-environment-variables/)

**Review by EneVladMihai — CHANGES_REQUESTED** [review]
Good work. Added two comments to bring the code in line with common industry practices.

**Review by Miladepour — APPROVED** [review]
I have finished my review.

**Pull request merged**